app/router.js

```diff
@@ -1,12 +1,37 @@
 import Ember from 'ember';
 import config from 'ember-get-config';
 
+const {hostname} = window.location;
+
+const provider = config
+    .PREPRINTS
+    .providers
+    // Exclude OSF
+    .slice(1)
+    // Filter out providers without a domain
+    .filter(p => p.domain)
+    .find(p =>
+        // Check if the hostname includes: the domain, the domain with dashes instead of periods, or just the id
+        hostname.includes(p.domain) ||
+        hostname.includes(p.domain.replace(/\./g, '-')) ||
```

Comment:
I had a conversation with @icereval a while back about how we might deploy this on staging/test. `agrixiv-org.staging.osf.io` was a likely naming scheme. Putting a dot in a subdomain creates another subdomain.